### ISSUE TITLE
Uninstall ember-cli-deploy

### ISSUE THREAD

**Issue by debragail** [ember-cli-deploy/ember-cli-deploy#496]
How do I uninstall?  It's breaking my build and I need to remove it.

**Comment by LevelbossMike** [comment]
To uninstall you can just use your node-package manager of choice (npm or yarn) to remove the package from package.json. E.g. `yarn remove ember-cli-deploy`. It seems odd that ember-cli-deploy would break your build though as its function is normally only executed on the `ember deploy`-command. Are you using the `runOnPostBuild`-functionality and are missing a dependency? -> http://ember-cli-deploy.com/docs/v1.0.x/development-workflow/

Closing this for now. Please feel free to contact us in the ember community Slack channel if you run into more trouble.

**Comment by debragail** [comment]
I'm just using git to push my build to heroku and i used npm uninstall to try to remove since i installed all the ember-cli-deploy add ons, but the dev dependencies first broke because it couldn't parse "bucket" so I added my s3 credentials and it was breaking on the the heroku buildpack.  I just needed to keep working on my project so I wanted to uninstall but the dev dependencies are still breaking my build.  I was still using bower because it's a legacy project so I"m moving into yarn now.  I plan on using AWS for one portion of the app though so I will probably use the deploy since i'll be running multiple environments on AWS.  Still waiting for yarn to finish installing ...
